modules/mocked_authentication/app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<html>
+  <%= yield %>
```

Comment:
Sooo if you want to get Rails technical (your favorite, I know). All your 'shared' stuff would go in here and you would yield in `main` or whatever, e.g:

```erb
<!DOCTYPE html>
<html>
<head>
  ...
</head>
<style>
  ...
</style>
<body>
  <noscript>
    ....
  </noscript>
  <header>
    ...
  </header>
  <main>
    <%= yield %>
  </main>
</body>
</html>
```
But whatever you want to do, especially since it's only one page... for now 🫡

Reply:
I tried the above way and remembered why I did it the way I did. I was unable to get the styles to load via the inline <style> tag.

Reply:
![oh-right-right](https://user-images.githubusercontent.com/10853209/233487900-69dcf8c2-2945-46f1-9a22-200022369f70.gif)

Reply:
You could probably put everything else in except the style but ¯\_(ツ)_/¯